src/engine/game/battle/enemybattler.lua

```diff
@@ -673,7 +671,6 @@ function EnemyBattler:defeat(reason, violent)
     end
     
     Game.battle.money = Game.battle.money + self.money
-    Game.battle.xp = Game.battle.xp + self.experience
```

Comment:
Same here.

Reply:
it isn't, but it seemed weird that you can gain EXP from sparing enemies, even though it's unused in Deltarune currently.
I can revert it if you want.

Reply:
It's only unused in Deltarune because XP is always set to 0. This works exactly like Deltarune does. Deltarune rules aren't necessarily the same as Undertale, so we shouldn't make assumptions in the engine as such.

Reply:
I see, I didn't know that. I will revert that change then.

Reply:
Done